flows/vision-zero/prefect_agent_environment_variables.env_template

```diff
@@ -44,3 +44,4 @@ EMS_S3_SOURCE_PREFIX=
 EMS_S3_SOURCE_PREFIX=
 
 DB_PERMISSION_SCHEMAS="import, hdb_catalog, public"
+DB_SSL_REQUIREMENT=required
```

Comment:
I think this needs to be updated to `require` 🔒

Reply:
Great catch, thank you! I didn't realize this was broken because I had adjusted it in my actual local env file.

I have settled on setting the suggested default for this to be 'prefer.' The idea is that when running locally, SSL/TLS isn't a concern, and further, the postGIS image we use (nor the one provided by the official postGIS folks) has support for SSL/TLS out of the box. It certainly can be made to support it, but we don't have a good use case for going through the trouble of providing legit certificates to make that happen. 

`prefer` is chosen because if someone does fire this up to communicate over the internet, hopefully the server will support SSL/TLS and the script will use it, but if not, such as when using this as a local service for development etc, it can fail to established an encrypted channel and still work. 

For our production deployment on `atd-data03`, we should stick with require, I think.